src/status_im/contexts/wallet/create_account/new_keypair/check_your_backup/view.cljs

```diff
@@ -62,26 +61,35 @@
 
 (defn- view-internal
   []
-  (let [random-indices  (random-selection)
-        quiz-index      (reagent/atom 0)
-        incorrect-count (reagent/atom 0)
-        show-error?     (reagent/atom false)]
+  (let [random-indices                        (random-selection)
+        quiz-index                            (reagent/atom 0)
+        incorrect-count                       (reagent/atom 0)
+        show-error?                           (reagent/atom false)
+        ;{:keys [secret-phrase random-phrase]} (rf/sub [:wallet/create-account])
+        {:keys [create-account]}              (rf/sub [:wallet/ui])
```

Comment:
this is still the wrong approach imo - 
we want to have a sub that does this, otherwise if any details within db :wallet :ui changes then this page re-renders. It's probably not such a big deal on this page in particular but something we want to avoid.  
```
{:keys [secret-phrase random-phrase]} (rf/sub [:wallet/create-account])
```
Apologies if this work is in flight, just commenting here to flag this for others too 👍

Reply:
@J-Son89 yeah I have this sub already `{:keys [secret-phrase random-phrase]} (rf/sub [:wallet/create-account])` but commented out to test something and forgot to reverse